adapters/Fyber/FyberAdapter/GADMAdapterFyberInitializer.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+/// Fyber adapter initialization completion handler.
+typedef void (^GADMAdapterFyberInitCompletionHandler)(NSError *_Nullable error);
+
+@interface GADMAdapterFyberInitializer : NSObject
```

Comment:
How does Fyber Marketplace SDK handle init in these situations:
1. Multiple simultaneous calls to init.
2. Called using different app IDs.

Reply:
1. In a case where the SDK is in the process of the init, we will ignore the next calls up until we will finish with the init.
2. If the SDK finished with the init and it gets a new init request, the SDK will check if there is something that needs to be updated, such as a new app ID. In this case, the SDK will fetch the relevant configuration.

Reply:
In case of (2), how will the Fyber Marketplace SDK handle it if a different App ID was used to initialize the SDK a second time? Will it overwrite the existing configuration?

Reply:
Yes, in this case, the SDK will use the new configuration only. It means that if the spotID's from the previous appID won't be valid anymore and the request will fail.

Reply:
We updated the PR to let the Fyber SDK handle the initialization.
Please take a look